content/blog/adoptium-reproducible-builds/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Reproducible Builds at Eclipse Adoptium
+date: "2022-03-31T12:00:00+00:00"
+author: anleonar@redhat.com
+description: Eclipse Adoptium is working towards fully reproducible builds, this blogs explains why
+ they are needed and the work ongoing to achieve them.
+tags:
+  - Temurin
+---
+
+## Why do we need Reproducible Builds?
+
+### Confidence in the high quality of binaries
+
+Being able to repeatably build the same OpenJDK source in an identical manner, producing
+identical JDK binaries, shows confidence and quality in the production process.
+The necessity to know exactly what is going into the binary and the exact repeatability of the
+build process, through what source and dependencies are used, to what compiler tooling is used.
+The fact that a reproducible identical JDK binary can only be achieved by all these stringent controls,
+shows the high quality of the JDK binaries to the consumers.
+
+### Secure Supply Chain
+
+The reproduciblity of builds goes hand in hand with knowing exactly what goes into it, in other words 
+the Software Bill of Materials (SBOM). This detailed knowledge means we know the source and provenance
+of everying within and used to build the JDK. Thus any publicly published CVEs or vulnerabilities can be
+quickly cross checked with the JDK and build tooling used.
+
+The other aspect of a secure supply chain is being able to absolutely identify a JDK binary as being secure,
+this is especially true for Open Source community projects. The ability to repeatably build a JDK binary
+identically from the Adoptium build scripts provides a mechanism to achieve this. The Adoptium releases can
+be re-built within multiple secure environments each with stringent bill of material checks, and then the binaries
+compared to ensure byte for byte identical output. This then gives the utmost secure supply chain confidence in
+the Adoptium community public binaries.
```

Comment:
Do you want to state it explicitly that "if the builds from the multiple secure environments are identical to the one from the open-source projects, the likelihood of malicious tampering would be extremely low".